chopper_tune.cfg

```diff
@@ -199,14 +204,27 @@ gcode:
             {% set minZAX = minAX %}
             {% set acceleration = settings.printer.max_z_accel|int %}
             {% set trv_speed = config.printer.max_z_velocity|int / 2 * 60 %} ; Idle movements speed
-        {% else %}
-            {action_raise_error('WARNING!!! Incorrect direction')}
+        {% elif axis != 'e'%}
+            {action_raise_error('WARNING!!! Incorrect direction', axis)}
```

Comment:
Only for delta kinematics

Reply:
yes but similar condition for other kinematics at #192